src/config.ts

```diff
@@ -49,6 +49,14 @@ export function envHead (): boolean {
 
 }
 
+export function envDisableExtra (): boolean {
+  const KEY = 'WECHATY_PUPPET_PUPPETEER_EXTRA_DISABLED'
+  return KEY in process.env
```

Comment:
I think we can just `return !!process.env[KEY]` at here. It's enough for this case.

At the same time, we can change the example from `WECHATY_PUPPET_PUPPETEER_DISABLE_EXTRA=true` to `WECHATY_PUPPET_PUPPETEER_DISABLE_EXTRA=1` to make it clearer.